source/initial-dns-seedlist-discovery.rst

```diff
@@ -0,0 +1,247 @@
+﻿.. role:: javascript(code)
+  :language: javascript
+
+==============================
+Initial DNS Seedlist discovery
+==============================
+
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
+:Spec Lead: Matt Broadstone
+:Advisory Group: A. Jesse Jiryu Davis
```

Comment:
The `A` needs to be escaped, otherwise rst decides this is a A. B. C. D. list

Reply:
FYI: `\A.` should do the trick.